backend/entityservice/settings.py

```diff
@@ -52,7 +53,10 @@ class Config(object):
     CELERY_RESULT_BACKEND_TRANSPORT_OPTIONS = {'master_name': "mymaster"} if REDIS_USE_SENTINEL else {}
 
     CELERY_ANNOTATIONS = {
-        'async_worker.calculate_mapping': {'rate_limit': '1/s'}
+        # Note that these are per worker instance per task rate limits.
```

Comment:
Is that why it was taking forever to empty the queue?

Reply:
No I don't think so because the `calculate_mapping` task was the only one rate limited and there actually isn't any task called that any more - hence the small correction to a correct path but commented out.